randovania/games/samus_returns/logic_database/Area 3 - Factory Exterior.txt

```diff
@@ -779,14 +818,15 @@ Extra - asset_id: collision_camera_038
   > Event - Gamma Metroid
       All of the following:
           Defeat Gamma Metroid
+          # Destroy Bomb Block
+          Lay Any Bomb
```

Comment:
Weird grouping in the DB (double `All of the following`)
![image](https://github.com/randovania/randovania/assets/38679103/99bb4430-f69e-48b4-bd54-ebd425fece95)

Reply:
necessary due to how the comment works.

Reply:
Could combine it with Break the blocks to reach the Gamma